From d4a15e93f1eccb5b39f55007c1b9037401a9d0ab Mon Sep 17 00:00:00 2001
From: Terry Lv <r65388@freescale.com>
Date: Thu, 29 Nov 2012 16:00:54 +0800
Subject: [PATCH 2299/2463] ENGR00235268: change caam_ipg_clk's CG to CG6

Another patch changed caam_ipg_clk's CG to CG4 and this commit will
revert this change.

Signed-off-by: Terry Lv <r65388@freescale.com>
---
 arch/arm/mach-mx6/clock.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx6/clock.c b/arch/arm/mach-mx6/clock.c
index fb4055c..0a8c880 100644
--- a/arch/arm/mach-mx6/clock.c
+++ b/arch/arm/mach-mx6/clock.c
@@ -4244,7 +4244,7 @@ static struct clk caam_clk[] = {
 	 __INIT_CLK_DEBUG(caam_ipg_clk)
 	.id = 2,
 	.enable_reg = MXC_CCM_CCGR0,
-	.enable_shift = MXC_CCM_CCGRx_CG4_OFFSET,
+	.enable_shift = MXC_CCM_CCGRx_CG6_OFFSET,
 	.enable = _clk_enable,
 	.disable = _clk_disable,
 	.parent = &mmdc_ch0_axi_clk[0],
-- 
1.7.7.4

